mlflow/recipes/utils/metrics.py

```diff
@@ -150,7 +150,7 @@ def _load_custom_metrics(recipe_root: str, metrics: List[RecipeMetric]) -> List[
         custom_metrics_mod = importlib.import_module("steps.custom_metrics")
         return [
             make_metric(
-                eval_fn=getattr(custom_metrics_mod, custom_metric.name),
+                eval_fn=getattr(custom_metrics_mod, custom_metric.custom_function),
```

Comment:
Found this issue when I was testing the classification template. We must use `custom_function`, not `name`.

```yaml
custom_metrics:
  # FIXME::OPTIONAL: Defines custom performance metrics to compute during model development.
    - name: mam
      function: my_awesome_metric_func
      greater_is_better: False
```

```python
def my_awesome_metric_func(). 
   return 1_000_000
```

Reply:
Should it be `custom_metric.function` ?

Reply:
`custom_metric` is a `RecipeMetric` object.

Reply:
Got it.

Reply:
If `custom_metric` was the correct one, tests would fail.